app/models/ticket_purchase.rb

```diff
@@ -58,4 +63,24 @@ def self.update_quantity(conference, quantity, ticket, user)
     purchase.quantity = quantity if quantity > 0
     purchase
   end
+
+  def pay(payment)
+    self.update_attributes(paid: true, payment_id: payment.id)
+  end
+
+  protected
+
+  def create_physical_tickets
```

Comment:
for example,

instead of setting paid to `true` in https://github.com/Ana06/osem/blob/7a7fcced487de914c9bd2bfc9e4e51839413aa8e/app/models/ticket_purchase.rb#L47

in the next line you can just do:

`purchase.pay if  ticket.price_cgents.zero?`